.gitignore

```diff
@@ -1,6 +1,7 @@
 build*/
 examples/**/postpro*/
 examples/**/log*/
+examples/**/*.log
```

Comment:
As none of the automation in the repo generates `.log` files, I'd be against this. This just seems to be happening if you are piping `stdout`/`stderr` to file (if you just put it into a `log/` directory like for the tests you will be fine.